src/pages/home/report/ReportActionsView.tsx

```diff
@@ -463,7 +463,7 @@ function ReportActionsView({
     }
 
     // AutoScroll is disabled when we do linking to a specific reportAction
-    const shouldEnableAutoScroll = hasNewestReportAction && (!reportActionID || !isNavigatingToLinkedMessage);
+    const shouldEnableAutoScroll = (hasNewestReportAction && (!reportActionID || !isNavigatingToLinkedMessage)) || reportActions.length === 1;
```

Comment:
Checking the report actions works well but feels a little workaround-ish. Can we instead use conditions that explicitly refer to the needed logic i.e. we have switched from many transactions view to one transaction view. Something like `transactionThreadReport && !prevTransactionThreadReport`